README.md

```diff
@@ -271,6 +273,25 @@ Full example:
         AppsFlyerLib.Shared.Start();
     }
 ```
+## ValidateAndLogV2
+[Here](https://dev.appsflyer.com/hc/docs/validate-and-log-purchase-ios) you can find the info on what is the ValidateAndLog API purpose.
+```c#
+AFSDKPurchaseDetails details = new AFSDKPurchaseDetails().initWithProductId("1234", "4.0", "USD", "123456789");
```

Comment:
The API was changed so we need to change the example as well:
`AFSDKPurchaseDetails details = new AFSDKPurchaseDetails("1234", "4.0", "USD", "123456789");`